src/js/components/Articles/ArticleReader.sass

```diff
@@ -0,0 +1,7 @@
+
+.Btn
+    margin: 0 auto
+    margin-top: 570px
```

Comment:
This is a very large top margin. What are you trying to achieve with it?

Reply:
i was trying to put the buttons in the bottom of the page,truly speaking I sucks with styling but i will learn scss

Reply:
I have new version of this repo with your comments and recommendation,thanks a lot